DTS-STN/Alpha-Site
# Feature/#126 Date Modified

# Description

[#126 Date Modified](https://trello.com/c/ys2VW2gK/126-126-date-modified-widget)

Canada.ca guidelines dictate that all pages must have a "Date Modified" component on the page which shows the date of the last change to the website/application in the common format 2020-04-20. You can see the guideline [here](https://design.canada.ca/common-design-patterns/date-modified.html).

This PR includes configuration to the Dockerfile to take in two new arguments: `BUILD_DATE` (the date of the build, passed from TeamCity in the format yyyyMMdd) and `TC_BUILD` (the build number of the build). These have been configured within Teamcity as well.

`BUILD_DATE` and `TC_BUILD` are passed into meta tags in the Head in _app.js. `BUILD_DATE` is passed into and used by the new DateModified component at build time. 

## Acceptance Criteria

There is a "date modified" widget with an ISO timestamp that updates based on an environment variable.

## Test Instructions

1. `npm run storybook`
2. navigate to Atoms/DateModified
3. see that component renders date and is translatable

1. `npm run dev`
2. go to homepage
3. see DateModified component renders above the footer
4. "NA" shows as date modified value because no date has been passed in

## Help Requested

- Anything you think could be improved

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
